### ISSUE TITLE
RB35S with X20pro issue got worse

### ISSUE THREAD

**Issue by noamkrief** [FrSkyRC/ETHOS-Feedback-Community#3042]
I turned stabilizing to off.
![rb35s](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/149995415/e2a01f85-9fbd-49c7-8ed9-d009cb8dce5a)

Now I cannot turn it back on. It is grayed out.

My communication with the RV35S is ok because I see current sensors in telemetry, and the config options are available.
But in the stab options, OFF is grayed out.

I've tried to rebind my TDXM and countless reboots of the radio and RB35S

attached proof.

PS - also unable to calibrate. Stuck on "top side up... waiting..."
Noam

SOLVED! 
After trying every conceivable setting for about 1 hour.
I changed the IMU mode from BASIC to OFF.
That didn't work.
I changed IMU to ADVANCED, and now I can enable Stab.

**Comment by bsongis-frsky** [comment]
Strange. I forward the issue.

**Comment by mawzthefinn** [comment]
That is odd. IMU mode to Off should do what you are experiencing (it disables the Gyro completely), but Basic is the correct IMU mode for the Lua-controlled stabilization. Advanced is supposed to be used with the RBMixer config app on Windows.

**Comment by noamkrief** [comment]
@mawzthefinn I cannot replicated the issue.
I can confirm my IMU was set to basic. 

How I finally figured it was related to IMU, is that in telemetry I had all RB35S sensors available, current sensors for 1-9 etc, but X Y Z were red!!!!

So I figured - "hey, it's talking to the RB35S, but something wrong with IMU data"

So then I turned IMU to off, then back to BASIC, and XYZ were still red. And turning on stab was still grayed out in OFF.

What got the whole thing to "wake up" is IMU to ADV.
Only after I put on ADV, I got XYZ back and stab option.

Now that everything is working normal, I can also confirm that:
Sometimes, when IMU = off I can still turn on/off stab. Sometimes when IMU = off, stab is grayed out.

I'm fairly certain that it's related to the chan 1-21 issue. Since 1-21 provides with "some" stability of the LUA, it's still will eventually stop communicating after some time and require a reboot of the RB35S.

So my hunch is that if communication happens to stop while "saving", it all goes haywire.

Thanks for these frsky employees + reps for monitoring these issues! Bugs happen, but customer service is KEY! 
Thank you!
